io.github.igorlogius.scr2ppm.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.github.igorlogius.scr2ppm
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: scr2ppm-wrapper
+rename-icon: scr2ppm
+rename-desktop-file: scr2ppm.desktop
+rename-appdata-file: scr2ppm.metainfo.xml
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=~/Screenshots:create
```

Comment:
Any reason it doesn't use a subdirectory in `xdg-pictures` instead?

Reply:
No reason, besides me not knowing that this was an option with `--filesystem`